app/Main.hs

```diff
@@ -1,6 +1,17 @@
 module Main where
 
-import Lib
+import Control.Monad (join)
+
+import Options.Applicative
```

Comment:
It's for command line parsing: https://robots.thoughtbot.com/applicative-options-parsing-in-haskell

I'm about to push some new code that looks a bit more like @pbrisbin's example